src/app/producer-shelf/components/tv2-action-card/tv2-action-card.component.html

```diff
@@ -1 +1,3 @@
-{{ action.name }}
+<span class="pt-1 ps-2 pe-2">{{ action.name }}</span>
+<div #titleContainer class="titleContainer"></div>
+<img *ngIf="isVideoClipAction() && videoClipThumbnailUrl" [ngSrc]="videoClipThumbnailUrl" [width]="getThumbnailWidth()" [height]="getThumbnailHeight()" [alt]="action.name" top>
```

Comment:
The `top` roguishly sneaked its way in. It has been removed in https://github.com/tv2/ng-sofie/pull/195